changelogs/fragments/181-ec2-group-info-python-fix.yaml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - Code fix in ec2_group_info so module works with Python 3.8 (make dict immutable in loop)
```

```suggestion
  - ec2_group_info - Code fix so module works with Python 3.8 (make dict immutable in loop) (https://github.com/ansible-collections/amazon.aws/pull/181)
```

Formatting / style niggle